src/modules/endpoint/DatabaseConnector.js

```diff
@@ -372,6 +374,8 @@ class DatabaseConnector {
         live: jsonObject.live,
         eventsEnabled: jsonObject.events_enabled,
         extrasEnabled: jsonObject.extras_enabled,
+        tunewsEnabled: false, // todo: NATIVE-549
```

Comment:
Strange TODOs :thinking:

Reply:
Well, these attributes have to be properly and carefully filled into the database. I didn't want to do this in this task.

Reply:
Was more like `todo` vs `TODO` but oc that's fine :smile: